src/config/mod.rs

```diff
@@ -24,6 +24,9 @@ pub struct DriaComputeNodeConfig {
     pub DKN_WALLET_ADDRESS: [u8; 20],
     /// Admin public key, used for message authenticity.
     pub DKN_ADMIN_PUBLIC_KEY: PublicKey,
+    /// Dria-searching-agent
```

Comment:
I would rather have the config of an optional task type be included within that task type alone, if possible.

If not, we may create a separate config for that and feature-gate it here under the main config